README.md

```diff
@@ -4,15 +4,44 @@ The Cluster Network Operator installs and upgrades the networking components on
 
 It follows the [Controller pattern](https://godoc.org/github.com/kubernetes-sigs/controller-runtime/pkg#hdr-Controller): it reconciles the state of the cluster against a desired configuration. The configuration specified by a CustomResourceDefinition called `networkoperator.openshift.io/NetworkConfig/v1`, which has a corresponding [type](/openshift/cluster-network-operator/blob/master/pkg/apis/networkoperator/v1/networkconfig_types.go).
 
+Most users will be able to use the top-level OpenShift Config API, which has a [Network type](https://github.com/openshift/api/blob/master/config/v1/types_network.go#L26). The operator will automatically translate the `Network.config.openshift.io` object in to a `NetworkConfig.networkoperator.openshift.io`.
+
 When the controller has reconciled and all its dependent resources have converged, the cluster should have an installed SDN plugin and a working service network. In OpenShift, the Cluster Network Operator runs very early in the install process -- while the boostrap API server is still running.
 
 # Configuring
-The network operator has a complex configuration, but most parameters have a sensible default.
+The network operator gets its configuration from two objects: the Cluster and the Operator configuration. Most users only need to create the Cluster configuration - the operator will generate its configuration automatically. If you need finer-grained configuration of your network, you will need to create both configurations.
+
+Any changes to the Cluster configuration are propagated down in to the Operator configuration.
+
+
+#### Configuration objects
+*Cluster config*
+- *Type Name*: `Network.config.openshift.io`
+- *Instance Name*: `cluster`
+- *View Command*: `oc get Network.openshift.io cluster -oyaml`
 
-The configuration must be called "default".
+*Operator config*
+- *Type Name*: `NetworkConfig.networkoperator.openshift.io`
+- *Instance Name*: `default`
+- *View Command*: `oc get NetworkConfig.networkoperator.openshift.io default -oyaml`
 
-A configuration with minimum parameters set:
+#### Example configurations
 
+*Cluster Config*
+```yaml
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  serviceNetwork: ["172.30.0.0/16"]
+  clusterNetwork: 
+    - cidr: "10.128.0.0/14"
+      hostPrefix: 23 
+  networkType: OpenShiftSDN
+```
+
+*Operator Config*
```

Comment:
(Clarify that the operator config in this example would have been auto-generated. Or alternatively, add a non-auto-generatable line to it and then explain that everything except that one line was auto-generated.)